acceptance/cmd/workspace/export-dir-file-size-limit/script

```diff
@@ -0,0 +1,7 @@
+echo "print('hello')" > test_file.py
```

Comment:
alternative you can just create this file in the acceptance test directory and commit it so you don't need to remove it below